src/settings/toml/dev.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct Dev {
+    pub host: Option<String>,
```

Comment:
im a bit wary of putting `host` in `wrangler.toml` because it's kinda half deprecated now because of edge dev...